signing_service/settings/__init__.py

```diff
@@ -1,6 +1,7 @@
-from .base import *
+from .base import *  # flake8: noqa
```

Comment:
"# flake8: noqa" tells flake8 to skip the whole file. "# noqa" tells it to skip a single line. I suspect that is what you meant to have occur here and below.